web/src/lib/components/album-page/albums-controls.svelte

```diff
@@ -0,0 +1,76 @@
+<script lang="ts" context="module">
+  import { writable } from 'svelte/store';
+
+  export const searchAlbum = writable<string>('');
+</script>
+
+<script lang="ts">
+  import LinkButton from '$lib/components/elements/buttons/link-button.svelte';
+  import Dropdown from '$lib/components/elements/dropdown.svelte';
+  import Icon from '$lib/components/elements/icon.svelte';
+  import { AlbumViewMode, albumViewSettings } from '$lib/stores/preferences.store';
+  import {
+    mdiArrowDownThin,
+    mdiArrowUpThin,
+    mdiFormatListBulletedSquare,
+    mdiPlusBoxOutline,
+    mdiViewGridOutline,
+  } from '@mdi/js';
+  import { sortByOptions, type Sort, handleCreateAlbum } from '$lib/components/album-page/albums-list.svelte';
+  import SearchBar from '$lib/components/elements/search-bar.svelte';
+
+  const searchSort = (searched: string): Sort => {
+    for (const key of sortByOptions) {
+      if (key.title === searched) {
+        return key;
+      }
+    }
+    return sortByOptions[0];
```

Comment:
I think that would be a good fit for a `sortByOptions.find((option) => option.title === searched) || sortByOptions[0]`